pkgs/development/lisp-modules-new-obsolete/ql.nix

```diff
@@ -1,99 +1,95 @@
 { pkgs, build-asdf-system, fixup ? pkgs.lib.id, ... }:
 
-with pkgs;
-with lib;
-with lib.lists;
-with lib.strings;
-
 let
+  inherit (pkgs) lib;
 
   # FIXME: automatically add nativeLibs based on conditions signalled
 
   # Try to keep this list sorted
   extras = {
     cffi-libffi = pkg: {
-      nativeBuildInputs = [ libffi ];
-      nativeLibs = [ libffi ];
+      nativeBuildInputs = [ pkgs.libffi ];
```

Comment:
Hm, I'd have done `extras = with pkgs; {` instead. Not sure this is worth the diff, as it's also adding noise now.